.nucleus.yaml

```diff
@@ -16,10 +16,7 @@ branches:
         - communities/ui-feeds-components
         - communities/ui-lightning-community
         - lwc/lwc-platform
-        - salesforce/lds-lightning-platform
```

Comment:
You could comment them out with a reference to the bug.

Reply:
Good idea. How about I start a Quip doc instead? I need to keep track of all of our downstreams that we're deliberately blocklisting.